relay-kafka/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license-file = "../LICENSE.md"
 publish = false
 
 [dependencies]
-rdkafka = { version = "0.29.0", optional = true, features = ["tracing"] }
+rdkafka = { version = "0.29.0", optional = true, features = ["tracing", "ssl"] }
```

Comment:
Not sure if we want to add this feature unconditionally. But since `rdkafka` is already optional itself, it does not matter much IMO.